tests/OpenToolkit.Tests.Integration/GameWindowTests.fs

```diff
@@ -105,6 +105,12 @@ module GameWindow =
             use gw = openGW()
             Assert.Equal(Vector2i(gw.Width, gw.Height), gw.ClientSize)
 
+        [<Fact>]
+        let ``Can query screen dpi`` () =
```

Comment:
You are really only testing if the screen dpi is never equal to zero
I suggest changing `Can query screen dpi` to `PPI is never zero`